versions/v6/header-front.php

```diff
@@ -5,6 +5,7 @@
 	</head>
 
 	<body <?php echo body_class( 'front-page' ); ?>>
+		<a class="skip-navigation bg-complementary text-inverse" href="#content">Skip to main content</a>
```

Comment:
We will need to add a placeholder div for the UCF Header immediately after this link so that it gets populated in afterward; otherwise the skip link won't be the first element in the `<body>` for keyboard users.  Here's an example: https://github.com/UCF/UCF-WordPress-Theme/blob/master/header.php#L7-L8